code-challenges.js

```diff
@@ -9,40 +9,149 @@
+    return timedFive
+}
 
+console.log(timesFive(myForNumbers1))
+console.log(timesFive(myForNumbers2))
 
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
-// Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
-var myForNumbers1 = [3, 7, 0, 6, -9]
-var myForNumbers2 = [2, 5, -8, 20, 32]
 
 
 
+// // --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
 
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
-
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+//create a function named multiFiveMap
+//takes in an array 
+const multiFiveMap = (array)=>{
+    // iterates through each index and creates a new array with the multiplied values
+    return array.map(value => {
+        //multiplies each value by five
+        return value * 5
+    })
+}
 
 
+console.log(multiFiveMap(myMapNumbers1))
+console.log(multiFiveMap(myMapNumbers2))
 
 
-// --------------------4) Create a function that removes all the vowels from a string.
-// Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
+// // --------------------4) Create a function that removes all the vowels from a string.
+// // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+//-----------------------------------------------------------------------------------------------------------
+//This was my original answer (#4 attempt one) This code worked, but if felt like a work around of what we had been doing in class.
+// I did a search with the google machine and found the ".replace(/[aeiou]/ig,'')" function. My understanding of this function is that it appears to identify the letters is the brackets and then replaces them with nothing which is why the '' is also in the parenthesis. Tested by putting 'R' into the space and a bunch of 'R's showed up in in the string
+
+// //Create a function called noVowel
+// //takes in an argument that contains a string
+// const noVowel = (vowels)=>{
+        //replaces the vowels space with nothing
+//     return vowels.replace(/[aeiouAEIOU]/ig,'')
+// }
+
+// I went back to tried to make this function based off of the tools learned this week in class (attempt two)
+// ------------------------------------------------------------------------------------------------------------
+//---------------4 attempt two)!!!!!!(Use this version)!!!!!!!
+
+//create a function and name it noVowelTwo
+//takes in an argument that is a string
+const noVowelTwo = (str) => {
```

Comment:
more descriptive function naming would be to start with a verb
```suggestion
const removeVowels = (str) => {
```